lambdas/postgres-migration-async-operation/package.json

```diff
@@ -11,15 +11,14 @@
   "types": "./dist/lambda/index.d.ts",
   "scripts": {
     "clean": "rm -rf dist",
-    "build": "rm -rf dist && mkdir dist && ../../node_modules/.bin/webpack",
```

Comment:
So I ran `npm run build` in this directory locally and got

```
RROR in Entry module not found: Error: Can't resolve './dist/lambda/index.js' in '/Users/mboyd/development/cumulus/lambdas/postgres-migration-async-operation'
```

And I think the cause is that the TS code isn't compiled before it starts to do the webpack build. If that is true, then we don't need to remove `build`, we just need to add `npm run tsc` before the webpack command

Reply:
Oh, interesting! I didn't realize the error was caused by the TS code not being compiled before the webpack build. Thanks!

Reply:
I just checked out the code locally and it fixes `npm run build` in this package, but I notice that the build in the API package also fails

```
ERROR in ./lambdas/sqs-message-consumer.js                                                                                              │ERROR in ./lambdas/sqs-message-consumer.js
Module not found: Error: Can't resolve '@cumulus/ingest/sqs' in '/Users/mboyd/development/cumulus/packages/api/lambdas'                 │Module not found: Error: Can't resolve '@cumulus/ingest/sqs' in '/Users/mboyd/development/cumulus/packages/api/lambdas'
 @ ./lambdas/sqs-message-consumer.js 25:4-34                                                                                            │ @ ./lambdas/sqs-message-consumer.js 25:4-34
                                                                                                                                        │npm ERR! code ELIFECYCLE
lerna ERR! npm run build stderr:                                                                                                        │npm ERR! errno 2
npm ERR! code ELIFECYCLE                                                                                                                │npm ERR! @cumulus/api@9.0.1 build: `../../node_modules/.bin/webpack`
npm ERR! errno 2                                                                                                                        │npm ERR! Exit status 2
npm ERR! @cumulus/api@9.0.1 build: `../../node_modules/.bin/webpack`                                                                    │npm ERR!
npm ERR! Exit status 2                                                                                                                  │npm ERR! Failed at the @cumulus/api@9.0.1 build script
```

I think the reason is that there is no `build` script in the `ingest` package, and the API package depends on being able to reference the compiled TS code in the `ingest` package. Should be as simple as adding a `build` script to the `ingest` package to run the TS compilation, but if you would rather not address in this PR I can do it as a follow up